WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -158,7 +158,6 @@ - (void)fetchPost:(NSUInteger)postID forSite:(NSUInteger)siteID isFeed:(BOOL)isF
 
     } failure:^(NSError *error) {
         if (failure) {
-            DDLogError(@"Error fetching post with id %@ and site %@. %@", postID, siteID, error);
```

Comment:
[not a blocker for merging] Since you're removing these logs, I'm not sure if this is worth mentioning, but is the app crashing because these values are scalars (`siteID` and `postID`) and the `%@` expects an object?
@twstokes mentioned this it ([on Slack](p1648053487743279/1648047079.017929-slack-C0JJ0C1UL), but not sure if you saw it).

Reply:
Hey @guarani . Per our discussion in Slack, I decided to just undo what I did for now and add proper logging later when I have a chance to do it right. Thank you!

Reply:
Sounds good!